SlowCheetah.JDT/JsonUtilities.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Utilities class for handling JSON files
+    /// </summary>
+    internal static class JsonUtilities
+    {
+        /// <summary>
+        /// The suffix for all JDT syntax
+        /// </summary>
+        internal const string JdtSyntaxPrefix = "@jdt.";
+
+        /// <summary>
+        /// Merges an array into another.
+        /// Merges a clone of the array.
+        /// </summary>
+        /// <param name="original">Array to merge into</param>
+        /// <param name="arrayToMerge">Array to be merged</param>
+        internal static void MergeArray(JArray original, JArray arrayToMerge)
+        {
+            foreach (JToken token in arrayToMerge)
+            {
+                original.Add(token.DeepClone());
+            }
+        }
+
+        /// <summary>
+        /// Loads a JSON file to a JToken
+        /// </summary>
+        /// <param name="filePath">Full path to the JSON file</param>
+        /// <returns>JObject extracted from the file</returns>
+        internal static JObject LoadObjectFromFile(string filePath)
+        {
+            if (string.IsNullOrEmpty(filePath))
+            {
+                throw new ArgumentNullException(nameof(filePath));
+            }
+
+            using (StreamReader file = File.OpenText(filePath))
+            using (JsonTextReader reader = new JsonTextReader(file))
+            {
+                JsonLoadSettings loadSettings = new JsonLoadSettings()
+                {
+                    CommentHandling = CommentHandling.Ignore,
+
+                    // Obs: LineInfo is handled on Ignore and not Load
```

Comment:
This is has been fixed, just need to wait for the next release of Newtonsoft to update the NuGet package. I'll still link to the issue though.